bundles/org.eclipse.e4.ui.css.swt.theme/src/org/eclipse/e4/ui/css/swt/internal/theme/Theme.java

```diff
@@ -43,10 +44,19 @@ public String getOsVersion() {
 		return this.osVersion;
 	}
 
+	@Override
+	public boolean IsDark() {
```

Comment:
It should be `isDark`.

Reply:
ah sure...

Reply:
see [e4d78b0](https://github.com/eclipse-platform/eclipse.platform.ui/commit/e4d78b04722d351282c43576242bf81758ecb7f6)